ShadowDOM/explainer.md

```diff
@@ -552,12 +736,16 @@ This suggestion looks like the following:
 
 ```html
 <my-element>
-   <template shadowrootmode="open" adoptedstylesheets="/foo.css">
-       <link rel="stylesheet" href="/foo.css" noadoptedstylesheets> <!-- no-op on browsers that support adoptedstylesheets on <template> tags -->
+   <template shadowrootmode="open" shadowrootadoptedstylesheets="/foo.css">
+       <link rel="stylesheet" href="/foo.css" noadoptedstylesheets> <!-- no-op on browsers that support shadowrootadoptedstylesheets on <template> tags -->
    </template>
 </my-element>
 ```
 
+## Future Work
```

Comment:
I wonder if we should rearrange stuff a bit so that the declarative HTML module etc stuff is also grouped in Future work?
That could also be done later.

Reply:
I have internal bug 58772693 assigned to me to refactor this in the near future - added this to the bug.